components/Accordion/Accordion.jsx

```diff
@@ -55,8 +55,6 @@ const AccordionHeader = styled.button.attrs({ type: 'button' })`
   display: inline-flex;
   justify-content: space-between;
   width: 100%;
-
-  ${({ as }) => (as === 'button' ? `cursor: pointer;` : null)}
```

Comment:
Why did you remove this?

Reply:
i want to know too rs

Reply:
Because when I went to see why this guy was not being covered in the unit tests, I notice that this prop is always undefined, and in line 54, there is a `cursor: pointer` already.